erpnext/startup/boot.py

```diff
@@ -75,3 +75,11 @@ def update_page_info(bootinfo):
 			"Sales Person Tree": {"title": "Sales Person Tree", "route": "Tree/Sales Person"},
 		}
 	)
+
+
+def bootinfo(bootinfo):
+	if bootinfo.get("user") and bootinfo["user"].get("name"):
+		bootinfo["user"]["employee"] = ""
+		employee = frappe.db.get_value("Employee", {"user_id": bootinfo["user"]["name"]}, "name")
```

Comment:
Use frappe.get_cached_value

Reply:
You can add filter in the frappe.get_cached_value
<img width="587" alt="image" src="https://github.com/frappe/erpnext/assets/8780500/993a09f6-1820-4c5a-96e7-0f7780860e21">

Reply:
Okay done.

This is very much not obvious from the get_cached_value code:

```py
def get_cached_value(
	doctype: str, name: str, fieldname: str = "name", as_dict: bool = False
) -> Any:
	try:
		doc = get_cached_doc(doctype, name)
```

Reply:
@rohitwaghchaure cached_value won't actually be cached in this case. It can only cache string names.